pymeasure/instruments/instrument.py

```diff
@@ -101,6 +102,35 @@ def id(self):
         else:
             raise NotImplementedError("Non SCPI instruments require implementation in subclasses")
 
+    @property
+    def quick_properties_list(self):
+        return self.quick_props_list
+
+    @quick_properties_list.setter
+    def quick_properties_list(self, prop_list):
+        if type(prop_list) is str:
+            self.quick_props_list = [prop_list]
+        elif type(prop_list) is list:
+            self.quick_props_list = prop_list
+        else:
+            raise TypeError("Expected input of type str or list")
+
+    @property
+    def quick_properties(self):
+        prop_dict = {}
+        for prop_name in self.quick_props_list:
+            prop_dict[prop_name] = getattr(self, prop_name)
+
+        return prop_dict
+
+    @quick_properties.setter
+    def quick_properties(self, setter_dict):
+        for key in setter_dict:
+            if key in self.quick_props_list:
+                setattr(self, key, setter_dict[key])
+            else:
+                raise RuntimeError("Property {} has not been placed in the quick properties list!".format(key))
```

Comment:
I agree with Casper here, on both counts. 
Another point to caution against a `quick_properties` property is that now we're "nesting" properties, we have properties acting on other properties, which introduces some mental complexity - up until now we could be ~sure that properties are instrument-related, which could be useful e.g. when enumerating an instrument's capabilities - with that suddenly there's this "meta"property which is not "like the others" (not sure if I convey that efficiently).

What do you think about this, instead:
`self.get_attributes(names=None)` queries one or more properties or attributes (getattr doesn't care). If `names` is not used, it falls back to a list `self.quick_attribute_list` as defined by the user.
`self.set_attributes(values)` sets the class attributes/properties as defined by the key-value pairs in the `values`.
Docstrings on both methods should explain the context and usage.

I think this achieves what you want, while at the same time being more generally useful, which is important because `Instrument` is the root of our driver hierarchy.